src/main/java/org/lobid/resources/ElasticsearchIndexer.java

```diff
@@ -91,6 +93,14 @@
 	public boolean lookupMabxmlDeletion;
 	/** Defines if a wikidata lookup should be done */
 	public boolean lookupWikidata;
+	/**
+	 * The date now. Handy to append to index-name to build multiple index' in
+	 * parallel. Switch then by setting the alias.
+	 */
+	public static final String DATE =
+			LocalDateTime.now().toLocalDate().format(DateTimeFormatter.BASIC_ISO_DATE)
+					+ "-" + LocalDateTime.now().getHour()
+					+ LocalDateTime.now().getMinute() + LocalDateTime.now().getSecond();
```

Comment:
Perhaps it would be safer to extract `LocalDateTime.now()` to a separate constant, otherwise we might get weird errors if the minute switches before we get the second or something like that.

Otherwise, or if that's not an issue, +1.